mod_reforged/config/item_names.nut

```diff
@@ -1,3 +1,52 @@
+// Vanilla Adjustments
+::MSU.Array.removeValues(::Const.Strings.CrossbowNames, [
+	"Betrayer",		// moved over to dagger names
+]);
+::Const.Strings.CrossbowNames.extend([
+	"Scorpion",
+	"Ironsight",
+	"Windlass",
+]);
+
+::MSU.Array.removeValues(::Const.Strings.DaggerNames, [
+	"Estoc",		// We have our own actual Kriegsmesser item
+]);
+::Const.Strings.DaggerNames.extend([
+	"Betrayer",		// formerly a crossbow name
+]);
+
+::Const.Strings.HandgonneNames.extend([
+	"Deafener",
+	"Eisenknall",
+	"Volcano",
+]);
+
+::Const.Strings.LongaxeNames.extend([
+	"Culler",
+]);
+
+::Const.Strings.SwordlanceNames.extend([	// In Vanilla this has only 2 entries
```

Comment:
Other cool swordlance names:
- Bloodstorm
- Warbringer
- Deathwind

Reply:
Warbringer sounds very generic. I added the other two though